apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Tests/SubscriberClientTest.cs

```diff
@@ -324,11 +324,7 @@ public override async Task AcknowledgeAsync(SubscriptionName subscription, IEnum
                 // Non-gRPC exceptions will propagate to the caller.
```

Comment:
Please modify line the comments above, as they're no longer accurate.

Reply:
Updated the comment.